adserver/views.py

```diff
@@ -137,24 +134,11 @@ class FlightListView(AdvertiserAccessMixin, UserPassesTestMixin, ListView):
 
     model = Flight
     template_name = "adserver/advertiser/flight-list.html"
-    PER_PAGE = 25
 
     def get_context_data(self, **kwargs):  # pylint: disable=arguments-differ
         context = super().get_context_data(**kwargs)
 
-        paginator = Paginator(self.get_queryset(), self.PER_PAGE)
-
-        # Note: Pagination has been greatly simplified in Django 2.x
-        try:
-            flight_list = paginator.page(self.request.GET.get("page"))
-        except PageNotAnInteger:
-            # If page is not an integer, deliver first page.
-            flight_list = paginator.page(1)
-        except EmptyPage:
-            # If page is out of range (e.g. 9999), deliver last page of results.
-            flight_list = paginator.page(paginator.num_pages)
-
-        context.update({"advertiser": self.advertiser, "flight_list": flight_list})
```

Comment:
Is there a reason we're removing pagination? Probably makes sense given there aren't that many flights, just curious.

Reply:
I removed pagination because we're breaking the flights up into 3 groups. I guess we could still paginate them but does that make sense in the context of groups?

Reply:
Yea, I think this is fine, we don't have anyone with hundreds of flights.